package.json

```
@@ -249,7 +249,7 @@
     "@metamask/address-book-controller": "^3.0.0",
     "@metamask/announcement-controller": "^4.0.0",
     "@metamask/approval-controller": "^3.4.0",
-    "@metamask/assets-controllers": "patch:@metamask/assets-controllers@npm%3A22.0.0#~/.yarn/patches/@metamask-assets-controllers-npm-22.0.0-6af3715fb2.patch",
+    "@metamask/assets-controllers": "23.0.0",
```

Could you use a `^` range here instead? We don't pin to specific versions unless we have to typically

```suggestion
    "@metamask/assets-controllers": "^23.0.0",
```